content/news/2023-01-25-foodborne-training/index.md

```diff
@@ -26,15 +34,70 @@ The workflow or the step by step tools used in the training can be summaried as
 
 ## Workshops
 
-- The [__first workshop__](https://galaxyproject.org/events/2023-02-02-foodborne-pathogen-detection-workshop/) conducted on the training was on the 2nd of Feburary 2023. 35 on-site and online participants from different background knowledge have attended the workshop. The workshop started with an introduction about Galaxy and the GTN, an introduction to the EOSC-Life industry call Foodborne pathogen detection project and finally our new GTN material was explained.
+The [__first workshop__](https://galaxyproject.org/events/2023-02-02-foodborne-pathogen-detection-workshop/) conducted on the training was on the 2nd of Feburary 2023. **35** on-site and online participants from different background knowledge have attended the workshop. The workshop started with an introduction about Galaxy and the GTN, an introduction to the EOSC-Life industry call Foodborne pathogen detection project and finally our new GTN material was explained.
+
+## Workshop Details
+We organized a practical workshop for (foodborne) pathogen detection from (direct Nanopore) sequencing data using Galaxy on February 2nd as a 1-day hybrid (F2F at FHNW Muttenz, Switzerland and online), free and practical workshop.
+
+More than **90 people registered**, 53% male, with diverse career stages and locations:
+
+![Participants Countries Graphs - Created by Bérénice Batut](/news/2023-01-25-foodborne-training/2023-01-25-workshop-participants-countries.png)
+
+![Participants Background Knowledge Graphs - Created by Bérénice Batut](/news/2023-01-25-foodborne-training/2023-01-25-workshop-participants-study-area.png)
```

Comment:
```suggestion
<img class="img-fluid float-right" src="/news/2023-01-25-foodborne-training/2023-01-25-workshop-participants-countries.png" style="width:300px;" alt="Participants Countries Graphs - Created by Bérénice Batut"/>

<img class="img-fluid float-right" src="news/2023-01-25-foodborne-training/2023-01-25-workshop-participants-study-area.png" style="width:300px;" alt="Participants Background Knowledge Graphs - Created by Bérénice Batut"/>
```

Reply:
Not sure it renders well. The idea is to have the images side by side. You will need to check locally how it looks like